test/test_sklearn_compat.py

```diff
@@ -56,6 +58,14 @@ def test_lsml(self):
   def test_itml(self):
     check_estimator(dITML)
 
+  # This fails in check_dtype_object, not sure why
+  # TODO !!!!!
```

Comment:
Any followup on this? I had to do some debugging when adding these tests initially, and it's not always obvious why they fail.